frontend/src/pages/Status.test.tsx

```diff
@@ -57,7 +57,7 @@ describe('Status', () => {
     });
 
     it('does not display any dates if neither was provided', () => {
-      const tree = shallow(statusToIcon(NodePhase.SUCCEEDED, /* No dates */));
+      const tree = shallow(statusToIcon(NodePhase.SUCCEEDED));
```

Comment:
why change this?

Reply:
vscode auto-format. I'll remove the comma, I think that's what's causing it to remove the comment.